spec/player_spec.rb

```diff
@@ -90,6 +92,36 @@
       end
     end
 
+    describe "#player_tiles" do
+      it "returns a collection of tiles that the player can play" do
+        expect(@player.player_tiles.length).to eq(0)
+        expect(@player.player_tiles).to be_an(Array)
+      end
+    end
+
+    describe "#draw_tiles(tile_bag)" do
+      before :all do
+        @playerJ = Scrabble::Player.new("J")
+        @tilebag2 = Scrabble::TileBag.new
+        @playerJ.player_tiles = ["d", "o", "g"]
+        @playerJ.draw_tiles(@tilebag2)
```

Comment:
I would recommend moving this line into the below `it` block on line 109, and also moving the `expect` from line 116 into that block as well.

From a conceptual standpoint, we're doing a single test action -- calling `draw_tiles` on a `Player` instance with a specific starting set of tiles. As a result of that action, we expect two different things to have happened: the `TileBag` instance has fewer tiles in it, and the `Player` instance's array of tiles has the same number of additional tiles in it.

IMO when we have a single action or sequence of actions but have multiple expectations about the results of those actions, it's best to include all of those `expect` lines within the same `it` block.